src/Namotion.Reflection/XmlDocsFormattingMode.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Namotion.Reflection
+{
+    /// <summary>
+    /// Contains the formatting modes supported.
+    /// </summary>
+    public enum XmlDocsFormattingMode
+    {
+        /// <summary>
+        /// Doesn't use any formatting.
+        /// </summary>
+        Unformatted,
```

Comment:
would `None` signal a better name, I haven't checked how other libraries like Newtonsoft names this

Reply:
Changed `Unformatted` to `None`. This is the same name that is also used in `Newtonsoft.Json.Formatting`.
Thanks for the hint!